server/src/index.js

```diff
@@ -1,5 +1,5 @@
 const express = require("express");
-const fs = require("fs");
+const getRepos = require("./api/getRepos");
```

Comment:
Try to separate the libraries from own packages by grouping them together and splitting the code with an empty line.